test/glue.js

```diff
@@ -0,0 +1,85 @@
+const tape = require('tape')
+const ssbKeys = require('ssb-keys')
+const Server = require('scuttle-testbot')
+const u = require('./util')
+
+const followedKeys = ssbKeys.generate()
+const blockedKeys = ssbKeys.generate()
+
+tape('listen to hopStream and requests replication of follows', function (t) {
```

Comment:
Mocks the replicate plugin to check that `sbot.replicate.request` is called when the glue/replicate file is activated, when a follow occurs.